mainboards/Makefile

```diff
@@ -20,73 +20,8 @@ docker-build: Dockerfile
 
 docker-run-aeeon-up: docker-build
 	docker run --rm -v "$(shell pwd)/aeeon/up":/$(CONTAINER_NAME) $(CONTAINER_NAME) 'make fetch flashkernel'
-
```

Comment:
let's remove this file entirely, it looks like it just creates lots of complexity - I hadn't even known about it

Reply:
looking at that Dockerfile, it *does* allow you to do some amount of testing in mainboards/Makefile.

If you are sure we can remove it, no problem though. Not sure the test is even meaningful at this point.

Reply:
since this breaks nothing, can we leave it here and address it in another PR?

Reply:
The docker support makes the Makefile complex as @orangecms said. We should have better approach if we want to reserve the docker support.